servers/audio_server.cpp

```diff
@@ -354,7 +354,7 @@ void AudioServer::_mix_step() {
 			playback->stream_playback->tag_used_streams();
 		}
 
-		if (mixed_frames != buffer_size) {
+		if (!playback->stream_playback->is_playing()) {
```

Comment:
I feel this is the wrong solution, and it should be fixed on the side of the generator code.